### PR TITLE
update the usage of tikv-ctl modify-tikv-config

### PR DESCRIPTION
**Pull request by NingLin-P**
Signed-off-by: linning <linningde25@gmail.com>

### What is changed, added or deleted? (Required)

This PR update the usage of `tikv-ctl modify-tikv-config`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

None

### CODE REVIEW

**Comment by NingLin-P** [comment]
PTAL @rleungx @qw4990

File: tikv-control.md [hunk 1]
```diff
@@ -327,56 +327,55 @@ tikv-ctl --db /path/to/tikv/data/db region-properties -r 2
 tikv-ctl --host 127.0.0.1:20160 region-properties -r 2
 ```
 
-### 动态修改 TiKV 的 RocksDB 相关配置
+### 动态修改 TiKV 的配置
 
-使用 `modify-tikv-config` 命令可以动态修改配置参数，暂时仅支持对于 RocksDB 相关参数的动态更改。
+使用 `modify-tikv-config` 命令可以动态修改配置参数，目前支持动态修改的配置与具体的行为与通过 SQL 动态修改配置功能相同，可参考[在线修改 TiKV 配置](/dynamic-config.md#在线修改-TiKV-配置)。
```
Comment (TomShawn) [inline tikv-control.md:332]:
```suggestion
使用 `modify-tikv-config` 命令可以动态修改配置参数。目前可动态修改的 TiKV 配置与具体的修改行为与 SQL 动态修改配置功能相同，可参考[在线修改 TiKV 配置](/dynamic-config.md#在线修改-TiKV-配置)。
```

File: tikv-control.md [hunk 2]
```diff
@@ -327,56 +327,55 @@ tikv-ctl --db /path/to/tikv/data/db region-properties -r 2
...
 
-- `-m` 用于指定要修改的模块，有 `storage`、`kvdb` 和 `raftdb` 三个值可以选择。
-- `-n` 用于指定配置名。配置名可以参考 [TiKV 配置模版](https://github.com/pingcap/tikv/blob/master/etc/config-template.toml#L213-L500)中 `[storage]`、`[rocksdb]` 和 `[raftdb]` 下的参数，分别对应 `storage`、`kvdb` 和 `raftdb`。同时，还可以通过 `default|write|lock + . + 参数名` 的形式来指定的不同 CF 的配置。对于 `kvdb` 有 `default`、`write` 和 `lock` 可以选择，对于 `raftdb` 仅有 `default` 可以选择。
+- `-n` 用于指定完整配置名。支持动态修改的配置名可以参考 [在线修改 TiKV 配置](/dynamic-config.md#在线修改-TiKV-配置)中支持的配置项列表。
```
Comment (TomShawn) [inline tikv-control.md:334]:
```suggestion
- `-n` 用于指定完整的配置名。支持动态修改的配置名可以参考[在线修改 TiKV 配置](/dynamic-config.md#在线修改-TiKV-配置)中支持的配置项列表。
```

File: tikv-control.md [hunk 3]
```diff
@@ -327,56 +327,55 @@ tikv-ctl --db /path/to/tikv/data/db region-properties -r 2
 tikv-ctl --host 127.0.0.1:20160 region-properties -r 2
 ```
 
-### 动态修改 TiKV 的 RocksDB 相关配置
+### 动态修改 TiKV 的配置
 
-使用 `modify-tikv-config` 命令可以动态修改配置参数，暂时仅支持对于 RocksDB 相关参数的动态更改。
+使用 `modify-tikv-config` 命令可以动态修改配置参数。目前可动态修改的 TiKV 配置与具体的修改行为与 SQL 动态修改配置功能相同，可参考[在线修改 TiKV 配置](/dynamic-config.md#在线修改-TiKV-配置)。
```
Comment (TomShawn) [inline tikv-control.md:332]:
```suggestion
使用 `modify-tikv-config` 命令可以动态修改配置参数。目前可动态修改的 TiKV 配置与具体的修改行为与 SQL 动态修改配置功能相同，可参考[在线修改 TiKV 配置](/dynamic-config.md#在线修改-tikv-配置)。
```

File: tikv-control.md [hunk 4]
```diff
@@ -327,56 +327,55 @@ tikv-ctl --db /path/to/tikv/data/db region-properties -r 2
...
 
-- `-m` 用于指定要修改的模块，有 `storage`、`kvdb` 和 `raftdb` 三个值可以选择。
-- `-n` 用于指定配置名。配置名可以参考 [TiKV 配置模版](https://github.com/pingcap/tikv/blob/master/etc/config-template.toml#L213-L500)中 `[storage]`、`[rocksdb]` 和 `[raftdb]` 下的参数，分别对应 `storage`、`kvdb` 和 `raftdb`。同时，还可以通过 `default|write|lock + . + 参数名` 的形式来指定的不同 CF 的配置。对于 `kvdb` 有 `default`、`write` 和 `lock` 可以选择，对于 `raftdb` 仅有 `default` 可以选择。
+- `-n` 用于指定完整的配置名。支持动态修改的配置名可以参考[在线修改 TiKV 配置](/dynamic-config.md#在线修改-TiKV-配置)中支持的配置项列表。
```
Comment (TomShawn) [inline tikv-control.md:334]:
```suggestion
- `-n` 用于指定完整的配置名。支持动态修改的配置名可以参考[在线修改 TiKV 配置](/dynamic-config.md#在线修改-tikv-配置)中支持的配置项列表。
```

**Pull request merged**